devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java

```diff
@@ -128,6 +132,18 @@ public void setUberJar(boolean uberJar) {
         this.uberJar = uberJar;
     }
 
+    @Optional
+    @Input
+    public List<String> getIgnoredEntries() {
+        return ignoredEntries;
+    }
+
+    @Option(description = "When using the uber-jar option, this option can be used to "
+            + "specify one or more entries that should be excluded from the final jar", option = "ignored-entry")
```

Comment:
perhaps change this option name to "ignored-entries"?

Reply:
Yes, I thought about that and I wasn't sure. 
The syntax when you specify these options in the command line is:

`./gradlew quarkusBuild --uber-jar --ignored-entry=META-INF/file1.txt`

If you want to specify more than 1 ignored entry, then you have to repeat the `--ignored-entry` argument:

`./gradlew quarkusBuild --uber-jar --ignored-entry=META-INF/file1.txt --ignored-entry=META-INF/file2.txt`

I used a singular word in the option because of the fact that you have to repeat it if you want to specify more than 1 entry. I'm fine with changing it to plural.

Reply:
ah, i thought it was a comma separated list or something, then its fine!